Populist Legal Power/common/laws/yMoG_PLP_citizenship.txt

```diff
@@ -408,6 +437,18 @@ law_multicultural = {
 	}
 	
 	acceptance_modifier = {
+		country_radicalism_increases_violent_hostility_mult   =  0.2
+		country_radicalism_increases_cultural_erasure_mult    =  0.15
+		country_radicalism_increases_open_prejudice_mult      =  0.1
+		country_radicalism_increases_second_rate_citizen_mult =  0.05
+		country_radicalism_increases_full_acceptance_mult     =  0
+		
+		country_loyalism_increases_violent_hostility_mult   = -0.1
+		country_loyalism_increases_cultural_erasure_mult    = -0.075
+		country_loyalism_increases_open_prejudice_mult      = -0.05
+		country_loyalism_increases_second_rate_citizen_mult = -0.025
+		country_loyalism_increases_full_acceptance_mult     =  0
```

Comment:
As above, literally everybody is at least a second rate citizen - consider contextual impacts maybe?